playground.ts

```diff
@@ -0,0 +1,27 @@
+import {HederaProvider} from "@ethersproject/providers";
```

Comment:
Lets create `examples` folder and place different `ts` files in that example folder. We would have `example` file for `providers.ts` f.e